os/board/esp32_devkit/src/spiram.c

```diff
@@ -103,17 +102,17 @@ bool esp_spiram_test()
 			}
 		}
 	}
-	ESP_EARLY_LOGI(TAG,"read spiram over\n");
+	lldbg("read spiram over\n");
```

Comment:
Let's use llvdbg. Because these logs are not errors.

Reply:
updated, use llvdbg here, and use dbg in test function